### PR TITLE
removes moths

### PR DESCRIPTION
**Pull request by SireTurret**
removes moths (no joke)

[Guidelines]: # 

## Changelog
:cl: Turret
del: Removed moths
/:cl:

## About The Pull Request
Removes moths, I promise it's not just a small change

## Why It's Good For The Game

because it will make saturniidae triggered

### CODE REVIEW

**Comment by notoriousBIGGIOS** [comment]
dab +1

**Comment by Saturniidae** [comment]
no please

**Comment by notoriousBIGGIOS** [comment]
DESTROYTHEFURRYMENACE

File: code/modules/mob/living/carbon/human/species_types/mothmen.dm [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/datum/species/moth
+/datum/species/moth //fuck mothfags everyone hates them
```
Comment (yoyobatty) [inline code/modules/mob/living/carbon/human/species_types/mothmen.dm:1]: This literally is just a comment in the code and doesn't do anything

**Pull request merged**